app/src/main/java/com/eveningoutpost/dexdrip/CalibrationOverride.java

```diff
@@ -51,7 +52,7 @@ public void onClick(View v) {
                     if (!TextUtils.isEmpty(string_value)){
                         final double calValue = JoH.tolerantParseDouble(string_value);
 
-                        Calibration last_calibration = Calibration.lastValid();
```

Comment:
I made this change, as well as all of the other missing upstream code.  I think the only diffs now between upstream server code and standalone-wear-2 are my changes.  I hope, that is!